Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```
@@ -213,8 +213,8 @@ script:
       secret: false
     - auto: PREDEFINED
       default: false
-      description: 'Ticket urgency. Giving a custom value which predefined in ServiceNow,
-        e.g: ''5 - Informational'' is available.'
+      description: 'Ticket urgency. You may select from the predefined options, or
+        define a custom field by using other value. e.g: "Urgent", "1 - Informational".'
```

@bakatzir @amshamah419 I think `custom field` term will confuse SNOW users. 
```suggestion
         use another value. e.g: "Urgent" or "5".'
```

---

Is it just "$Label" or can it be both "$INT - $Label"?